sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/DiskInspectionOperationsExtensions.cs

```diff
@@ -1,143 +0,0 @@
-// <auto-generated>
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// </auto-generated>
-
-namespace Microsoft.Azure.Management.Compute
-{
-    using Microsoft.Rest;
-    using Microsoft.Rest.Azure;
-    using Models;
-    using System.Threading;
-    using System.Threading.Tasks;
-
-    /// <summary>
-    /// Extension methods for DiskInspectionOperations.
-    /// </summary>
-    public static partial class DiskInspectionOperationsExtensions
-    {
-            /// <summary>
-            /// Creates a request for executing disk Inspection.
-            /// </summary>
-            /// <param name='operations'>
-            /// The operations group for this extension method.
-            /// </param>
-            /// <param name='location'>
-            /// The location used to execute the diagnostic operation.
-            /// </param>
-            /// <param name='runDiskInspectionInput'>
-            /// RunDiskInspection object supplied in the body of the Post run disk
-            /// inspection operation.
-            /// </param>
-            public static string Create(this IDiskInspectionOperations operations, string location, RunDiskInspectionInput runDiskInspectionInput)
```

Comment:
I'm wonder why we don't have to provide customization for these deleted methods such as this one. Is it because they are just from the last preview release?

Reply:
yeah they were just for the preview release